include/config/overworld.h

```diff
@@ -13,6 +13,7 @@
 #define OW_PC_PRESS_B               GEN_LATEST // In Gen4, pressing B when holding a Pokémon is equivalent to placing it. In Gen3, it gives the "You're holding a Pokémon!" error.
 #define OW_PC_JAPAN_WALDA_ICONS     TRUE       // In the US release of Emerald, the Cross, Bolt, and Plusle icons for Walda's wallpapers were left blank from the Japan release. Setting this to TRUE will restore them.
 #define OW_PC_HEAL                  GEN_LATEST // In Gen8+, Pokémon are not healed when deposited in the PC.
+#define OW_PC_MOVE_ORDER            GEN_LATEST // In Gen4+, Move Pokémon becomes the first option in the PC Menu.
```

Comment:
This is not the case. It was ORAS that even introduced the option to put "Move" on top, and from Gen 7 onwards it automatically opens it.
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/af4c9a4b-5843-4124-a900-1f830705cdb6)

Reply:
Ah! SOMETHING changed in Gen 4, but I got it wrong. Thanks for the reminder, now fixed.